packages/volto/src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -600,19 +614,33 @@ class Toolbar extends Component {
                 {!this.props.hideDefaultViewButtons && (
                   <button
                     className="user"
-                    aria-label={this.props.intl.formatMessage(
-                      messages.personalTools,
-                    )}
+                    aria-label={
+                      userHasRoles(this.props.user, [
+                        'Site Administrator',
+                        'Manager',
+                      ])
+                        ? this.props.intl.formatMessage(
+                            messages.adminUserlTools,
+                          )
+                        : this.props.intl.formatMessage(messages.userTools)
+                    }
                     onClick={(e) => this.toggleMenu(e, 'personalTools')}
                     tabIndex={0}
                     id="toolbar-personal"
                   >
                     <Icon
                       name={userSVG}
                       size="30px"
-                      title={this.props.intl.formatMessage(
-                        messages.personalTools,
-                      )}
+                      title={
+                        userHasRoles(this.props.user, [
+                          'Site Administrator',
+                          'Manager',
+                        ])
+                          ? this.props.intl.formatMessage(
+                              messages.adminUserlTools,
+                            )
+                          : this.props.intl.formatMessage(messages.userTools)
```

Comment:
Can you make the message a variable so it is easier to keep in-sync with the above button?